packages/docs/src/pages/en/getting-started/upgrade-guide.md

```diff
@@ -204,3 +204,7 @@ app.use(vuetify)
 ### v-snackbar
 
 - `action` slot was renamed to `actions`
+
+### v-expansion-panel
+
+- `v-expansion-panel-header` has been renamed to `v-expansion-panel-title` and `v-expansion-panel-content` into `v-expansion-panel-text` also you can use new props in `v-expansion-panel` `text` and `title`.
```

Comment:
Separate bullet points:
```suggestion
- `v-expansion-panel-header` has been renamed to `v-expansion-panel-title`.
- `v-expansion-panel-content` has been renamed to `v-expansion-panel-text`.
- `v-expansion-panel` now has `text` and `title` props that can be used instead of subcomponents.
```